spec/search_builder/sufia/my_shares_search_builder_spec.rb

```diff
@@ -11,8 +11,8 @@
   subject { builder.to_hash['fq'] }
 
   it "filters things we have access to in which we are not the depositor" do
-    expect(subject).to eq ["edit_access_group_ssim:public OR edit_access_group_ssim:registered OR edit_access_person_ssim:#{me.user_key}",
+    expect(subject).to eq ["({!terms f=edit_access_group_ssim}public,registered) OR edit_access_person_ssim:user1@example.com",
                            "{!terms f=has_model_ssim}GenericWork,Collection",
-                           "-_query_:\"{!field f=depositor_ssim}#{me.user_key}\""]
+                           "-_query_:\"{!field f=depositor_ssim}user1@example.com\""]
```

Comment:
I don't think you want to hard code the username. Continue to use `#{me.user_key}`